modules/core/client/views/sidebar.client.view.html

```diff
@@ -0,0 +1,40 @@
+<section id="sidebar">
+    <br>
+    <br>
+    <br>
+    <div ng-controller="sidebarCtrl">
```

Comment:
!fix Should be declared in the section tag at the top of the html content, where id="sidebar" is currently defined